src/PowerAppsTestEngine/Program.cs

```diff
@@ -178,7 +178,11 @@
 
         //setting defaults for optional parameters outside RunTestAsync
         var testResult = await testEngine.RunTestAsync(testPlanFile, environmentId, tenantId, outputDirectory, domain, queryParams);
-        Console.WriteLine($"Test results can be found here: {testResult}");
+        if (testResult != "InvalidOutputDirectory")
```

Comment:
In PAC CLI >> when we update the TE version with this fix included, reminder to make this change there.